idd/Energy+.idd.in

```diff
@@ -20878,15 +20884,45 @@ People,
        \key KSU
        \key AdaptiveASH55
        \key AdaptiveCEN15251
+       \key CoolingEffectASH55
+       \key AnkleDraftASH55
        \note optional (fourth thermal comfort model and report type)
-  A18; \field Thermal Comfort Model 5 Type
+  A18, \field Thermal Comfort Model 5 Type
        \type choice
        \key Fanger
        \key Pierce
        \key KSU
        \key AdaptiveASH55
        \key AdaptiveCEN15251
+       \key CoolingEffectASH55
+       \key AnkleDraftASH55
        \note optional (fifth thermal comfort model and report type)
+  A19, \field Thermal Comfort Model 6 Type
+       \type choice
+       \key Fanger
+       \key Pierce
+       \key KSU
+       \key AdaptiveASH55
+       \key AdaptiveCEN15251
+       \key CoolingEffectASH55
+       \key AnkleDraftASH55
+       \note optional (fifth thermal comfort model and report type)
+  A20, \field Thermal Comfort Model 7 Type
+       \type choice
+       \key Fanger
+       \key Pierce
+       \key KSU
+       \key AdaptiveASH55
+       \key AdaptiveCEN15251
+       \key CoolingEffectASH55
+       \key AnkleDraftASH55
+       \note optional (fifth thermal comfort model and report type)
+  A21; \field Ankle Level Air Velocity Schedule Name
```

Comment:
Hmmm, this new field would work better if it was moved up to be A14 before Thermal Comfort Model 1 Type. But that would add a transition requirement. Since the thermal comfort model fields are not extensible (because there's a limited set of options). So, I guess it's fine here.

Reply:
Yes. I put it at the bottom as too many transitions change would be triggered to add a field in the middle of `People` object.